bin/etest

```diff
@@ -134,9 +142,10 @@ fi
 START_TIME=${SECONDS}
 
 # Default log directory from conf file if unspecified on the command line
-: ${log_dir:=$(conf_get _EBASH_CONF etest.log_dir)}
-: ${log_dir:=.}
-log_dir=$(readlink -f ${log_dir})
+: ${logdir:=$(conf_get _EBASH_CONF etest.log_dir)}
+: ${logdir:=$(conf_get _EBASH_CONF etest.logdir)}
```

Comment:
These two lines allow us to use the older `log_dir` and the newer `logdir` to remain backwards compatible